tutorindigo/templates/indigo/lms/templates/static_templates/about.html

```diff
@@ -8,7 +8,7 @@
 <main id="main" aria-label="Content" tabindex="-1">
     <section class="container about">
         <h1>
-            <%block name="pageheader">${page_header or _("About")}</%block>
+            <%block name="pageheader">${page_header or _("About Us")}</%block>
```

Comment:
Are we sure we want to make that change? "About us" sounds more appropriate for small platforms. "About" is more anonymous, which we want (I think). Is this change consistent with the rest of the theme?

Reply:
About is fine and we can go with it. This change was only consistent with footer links label "about us" and "contact us" which I believe user can change using config.